### PR TITLE
completed get/api/articles returning in date desc order #9

### PR DESCRIPTION
**Pull request by kam-cheng**
completed get/api/articles returning in date desc order #9

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -14,14 +14,46 @@ describe("GET/api/topics", () => {
...
+  test("200 - returns 200 Status", async () => {
+    await request(app).get("/api/articles").expect(200);
+  });
+  test("200 - return array of the correct length", async () => {
+    const articles = await request(app).get("/api/articles").expect(200);
+    expect(articles.body).toHaveLength(12);
+  });
+  test("200 - returns array of article objects with numerous properties", async () => {
+    const articles = await request(app).get("/api/articles").expect(200);
+    console.table(articles.body);
```
Comment (Angaca) [inline __tests__/app.test.js:41]: Remember to remove console.log from production code

**Review by Angaca — CHANGES_REQUESTED** [review]
Be careful you should send everything in a specific key -> like {topics} and {articles} so you can easily reference to them. You should change your test accordingly

File: __tests__/app.test.js (hunk 1 above)
Comment (kam-cheng) [reply __tests__/app.test.js]: thanks - have now removed :)

File: controllers/app.controllers.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
 
 exports.getTopics = async (req, res) => {
   const topics = await fetchTopics();
   res.status(200).send(topics);
 };
+
+exports.getArticles = async (req, res) => {
+  const articles = await fetchArticles();
+  res.status(200).send(articles);
+};
```
[the remark this answers is not in the document]
Comment (kam-cheng) [reply controllers/app.controllers.js:11]: cheers! getting some practice in after last Friday

**Review by Angaca — APPROVED** [review]
Good to go now!

**Pull request merged**